server/entity/MediaRequest.ts

```diff
@@ -377,6 +378,7 @@ export class MediaRequest {
     eager: true,
     onDelete: 'CASCADE',
   })
+  @JoinColumn()
```

Comment:
Does this not make a change to the database? Not sure why this wasn't needed until now?

Reply:
It shouldn't be necessary if specify the relation when we try to find the correct media. Will update in the code.